docs/usage.md

```diff
@@ -6,11 +6,15 @@ The SEO tag will respect any of the following if included in your site's `_confi
 * `description` - A short description (e.g., A blog dedicated to reviewing cat gifs)
 * `url` - The full URL to your site. Note: `site.github.url` will be used by default.
 * `author` - global author information (see below)
-* `twitter:username` - The site's Twitter handle. You'll want to describe it like so:
+
+* `twitter` - The following properties are available:
+  * `twitter:username` - The site's Twitter handle
+  * `twitter:card` - The site's default card type
 
   ```yml
   twitter:
     username: benbalter
+    card: summary_large_image
```

Comment:
What are the other card types available? We could use a value different from the default's one here.